mysql/dbstat/dbstat.go

```diff
@@ -37,39 +37,38 @@ type MysqlStatMetrics struct {
 	RelayLogSpace            *metrics.Gauge
 
 	//GetGlobalStatus
-	AbortedConnects           *metrics.Counter
-	BinlogCacheDiskUse        *metrics.Counter
-	BinlogCacheUse            *metrics.Counter
-	ComAlterTable             *metrics.Counter
-	ComBegin                  *metrics.Counter
-	ComCommit                 *metrics.Counter
-	ComCreateTable            *metrics.Counter
-	ComDelete                 *metrics.Counter
-	ComDeleteMulti            *metrics.Counter
-	ComDropTable              *metrics.Counter
-	ComInsert                 *metrics.Counter
-	ComInsertSelect           *metrics.Counter
-	ComReplace                *metrics.Counter
-	ComReplaceSelect          *metrics.Counter
-	ComRollback               *metrics.Counter
-	ComSelect                 *metrics.Counter
-	ComUpdate                 *metrics.Counter
-	ComUpdateMulti            *metrics.Counter
-	CreatedTmpDiskTables      *metrics.Counter
-	CreatedTmpFiles           *metrics.Counter
-	CreatedTmpTables          *metrics.Counter
-	InnodbCurrentRowLocks     *metrics.Gauge
-	InnodbLogOsWaits          *metrics.Gauge
-	InnodbRowLockCurrentWaits *metrics.Gauge
-	InnodbRowLockTimeAvg      *metrics.Gauge
-	InnodbRowLockTimeMax      *metrics.Counter
-	PreparedStmtCount         *metrics.Gauge
-	PreparedStmtPct           *metrics.Gauge
-	Queries                   *metrics.Counter
-	SortMergePasses           *metrics.Counter
-	ThreadsConnected          *metrics.Gauge
-	Uptime                    *metrics.Counter
-	ThreadsRunning            *metrics.Gauge
+	AbortedConnects      *metrics.Counter
+	BinlogCacheDiskUse   *metrics.Counter
+	BinlogCacheUse       *metrics.Counter
+	ComAlterTable        *metrics.Counter
+	ComBegin             *metrics.Counter
+	ComCommit            *metrics.Counter
+	ComCreateTable       *metrics.Counter
+	ComDelete            *metrics.Counter
+	ComDeleteMulti       *metrics.Counter
+	ComDropTable         *metrics.Counter
+	ComInsert            *metrics.Counter
+	ComInsertSelect      *metrics.Counter
+	ComReplace           *metrics.Counter
+	ComReplaceSelect     *metrics.Counter
+	ComRollback          *metrics.Counter
+	ComSelect            *metrics.Counter
+	ComUpdate            *metrics.Counter
+	ComUpdateMulti       *metrics.Counter
+	CreatedTmpDiskTables *metrics.Counter
+	CreatedTmpFiles      *metrics.Counter
+	CreatedTmpTables     *metrics.Counter
+	InnodbLogOsWaits     *metrics.Gauge
+	InnodbRowLockWaits   *metrics.Gauge
+	InnodbRowLockTimeAvg *metrics.Gauge
+	InnodbRowLockTimeMax *metrics.Counter
```

Comment:
InnodbRowLockTimeMax should probably be a gauge - because it is measure of something at a point in time not necessarily something we want to calculate rate on.